### PR TITLE
[ZH] Update `Internet Relay Chat`

### PR DESCRIPTION
**Pull request by cdwcgt**
翻译中保留了一些英文，因为在用户界面上没有中文翻译
Some English are kept in the translation as there is no Chinese translation on the UI

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Community/Internet_Relay_Chat/zh.md [hunk 1]
```diff
@@ -1,88 +1,87 @@
...
-1. 检查你使用的密码是否和 [IRC Authentication](https://osu.ppy.sh/p/irc) 页面上的一致。
-2. 如果你的用户名中有空格，用下划线替代它 (**This Username** 应该是 **This\_Username**)
+### 为什么有些人的用户名前缀不同？
 
-### 我能使用其他的昵称么?
+IRC 标准有一个 IRC 模式的概念，可以理解为每个用户的权限或状态。在 osu!Bancho 中，有两种模式被用于特殊的用户组
 
-不能，你必须使用你游戏中的用户名。
+- `+`，或者“语音状态”：用户通过外部 IRC 客户端连接
+- `@`，或者“聊天管理员状态”：用户是聊天室管理员（[GMT](/wiki/People/The_Team/Global_Moderation_Team) 或者 [NAT](/wiki/People/The_Team/Nomination_Assessment_Team)）
```
Comment (YumeMuzi) [inline wiki/Community/Internet_Relay_Chat/zh.md:81]: 虽然没有经过讨论，但是我将会采用这两个词的新翻译，所以这两个词最好改成：[GMT（全局管理团队）](/wiki/People/The_Team/Global_Moderation_Team)、[NAT（审核评估团队）](/wiki/People/The_Team/Nomination_Assessment_Team)

File: wiki/Community/Internet_Relay_Chat/zh.md [hunk 2]
```diff
@@ -1,88 +1,87 @@
...
-someuser has quit #somechannel
-```
+或者你可以尝试连接到 `cho.ppy.sh`（你仍然会连接至 osu!Bancho）
 
-尽管在某些低流量的频道中这不是什么，但对于 \#osu 频道来说，频繁的加入和退出通知会让用户难以追踪最近的消息。
+### 我收到了 "Bad Authentication Token" 错误 <!-- Bad Authentication Token是由 Bancho 发送的固定字符串，与客户端语言设置无关，故保留英文 -->
 
-### 操作方法
+1. 检查你使用的密码是否和 [IRC 验证页面](https://osu.ppy.sh/p/irc) 上的一致。
+2. 如果你的用户名中有空格，用下划线替代它（例如，`This Username` 应该是 `This_Username`）
```
Comment (YumeMuzi) [inline wiki/Community/Internet_Relay_Chat/zh.md:66]:
```suggestion
2. 应使用下划线替代用户名中的空格（例如，`This Username` 应按 `This_Username` 的格式来输入）。
```

File: wiki/Community/Internet_Relay_Chat/zh.md [hunk 3]
```diff
@@ -1,88 +1,87 @@
...
+osu!Bancho 使用纯文本身份验证——确保在您的 IRC 客户端设置中没有选择特殊的身份验证模式。
 
-```
-someuser has joined #somechannel
-someuser has quit #somechannel
-```
+或者你可以尝试连接到 `cho.ppy.sh`（你仍然会连接至 osu!Bancho）
 
-尽管在某些低流量的频道中这不是什么，但对于 \#osu 频道来说，频繁的加入和退出通知会让用户难以追踪最近的消息。
+### 我收到了 "Bad Authentication Token" 错误 <!-- Bad Authentication Token是由 Bancho 发送的固定字符串，与客户端语言设置无关，故保留英文 -->
```
Comment (YumeMuzi) [inline wiki/Community/Internet_Relay_Chat/zh.md:63]:
```suggestion
### 我收到了 "Bad Authentication Token" （身份验证令牌）错误
```

File: wiki/Community/Internet_Relay_Chat/zh.md [hunk 4]
```diff
@@ -1,88 +1,87 @@
...
 
-不能，你必须使用你游戏中的用户名。
+- `+`，或者“语音状态”：用户通过外部 IRC 客户端连接
+- `@`，或者“聊天管理员状态”：用户是聊天室管理员（[GMT](/wiki/People/The_Team/Global_Moderation_Team) 或者 [NAT](/wiki/People/The_Team/Nomination_Assessment_Team)）
 
-### 我在某些人的名字前看到了 voice 状态，那是什么？
+通过游戏或网站连接到 osu!Bancho 的用户将没有用户名前缀。
 
-voice 状态（名字前面有一个“+”）的用户是通过 IRC 客户端连接的用户，（除了 IRC 管理员，他们的名字前有一个“@”）。
+### 有人给我发送了私信，但是他们并没有在频道用户列表中
```
Comment (YumeMuzi) [inline wiki/Community/Internet_Relay_Chat/zh.md:85]:
```suggestion
### 有人给我发送了私信，但是他们并没有在频道用户列表中！
```
前面的地方好像也有标点符号，你检查一下

File: wiki/Community/Internet_Relay_Chat/zh.md [hunk 5]
```diff
@@ -1,88 +1,87 @@
...
 
-### 在 Bancho 上验证
+- 服务器地址: `irc.ppy.sh`
+- 服务器端口: `6667`
+- SSL（安全套接字）: disabled （禁用）
+- 用户名：你的 osu! 用户名。使用下划线代替空格（例如，`beppy master 1000` 应该填写为 `beppy_master_1000`）
+- 密码：从 [IRC 验证页面](https://osu.ppy.sh/p/irc)获取
 
-当你连接上时会弹出下面的消息。
+*警告：你的 IRC 密码不同于你的账户密码。**不要把你的密码分享给别人**。*
```
Comment (YumeMuzi) [inline wiki/Community/Internet_Relay_Chat/zh.md:23]:
```suggestion
*警告：你的 IRC 密码与账户密码不同。**不要把你的密码分享给别人**。*
```

File: wiki/Community/Internet_Relay_Chat/zh.md [hunk 6]
```diff
@@ -1,88 +1,87 @@
----
-outdated_translation: true
----
-
 # 互联网中继聊天
 
-[互联网中继聊天](https://zh.wikipedia.org/wiki/IRC), 简称 IRC, 是一种用于群体聊天的协议。
+*如果要了解更多，请参见[互联网中继聊天](https://baike.baidu.com/item/%E5%9B%A0%E7%89%B9%E7%BD%91%E4%B8%AD%E7%BB%A7%E8%81%8A%E5%A4%A9)<!-- 这里使用百度百科的内容，因为中国大陆无法访问维基百科 The content of Baidu Baike is used here, because Wikipedia cannot be accessed in mainland China-->
```
Comment (YumeMuzi) [inline wiki/Community/Internet_Relay_Chat/zh.md:3]: 如果维基百科上有，还是换成维基百科吧，百度百科的内容基本上不全

**Review by YumeMuzi — CHANGES_REQUESTED** [review]
Only part of the content needs to be fixed, others seem fine

File: wiki/Community/Internet_Relay_Chat/zh.md (hunk 6 above)
Comment (cdwcgt) [reply wiki/Community/Internet_Relay_Chat/zh.md:3]: 那这里暂时先使用维基
之后拉个投票

File: wiki/Community/Internet_Relay_Chat/zh.md [hunk 7]
```diff
@@ -1,88 +1,87 @@
...
 
-### 我能使用其他的昵称么?
+IRC 标准有一个 IRC 模式的概念，可以理解为每个用户的权限或状态。在 osu!Bancho 中，有两种模式被用于特殊的用户组
 
-不能，你必须使用你游戏中的用户名。
+- `+`，或者“语音状态”：用户通过外部 IRC 客户端连接
+- `@`，或者“聊天管理员状态”：用户是聊天室管理员（[GMT（全局管理团队）](/wiki/People/The_Team/Global_Moderation_Team) 或者 [NAT（审核评估团队）](/wiki/People/The_Team/Nomination_Assessment_Team)）
 
-### 我在某些人的名字前看到了 voice 状态，那是什么？
+通过游戏或网站连接到 osu!Bancho 的用户将没有用户名前缀。
```
Comment (cl8n) [inline wiki/Community/Internet_Relay_Chat/zh.md:81]: seems different than EN here

File: wiki/Community/Internet_Relay_Chat/zh.md [hunk 8]
```diff
@@ -1,88 +1,87 @@
...
 
-voice 状态（名字前面有一个“+”）的用户是通过 IRC 客户端连接的用户，（除了 IRC 管理员，他们的名字前有一个“@”）。
+### 有人给我发送了私信，但是他们并没有在频道用户列表中！
 
-名字前没有前缀的用户是通过游戏连接到 osu! Bancho 的用户。
+他们要么在使用[网页聊天](https://osu.ppy.sh/community/chat)或者使用[osu!(lazer)](https://github.com/ppy/osu)来连接 osu!Bancho。
```
Comment (cl8n) [inline wiki/Community/Internet_Relay_Chat/zh.md:85]: same as above, not sure what the mention of osu!bancho is about

File: wiki/Community/Internet_Relay_Chat/zh.md [hunk 9]
```diff
@@ -1,88 +1,85 @@
...
-| [ircII](http://www.eterna.com.au/ircii/) | `/ignore * crap` |
-| [Irssi](https://irssi.org) | `/ignore -channels #频道名称 * JOINS PARTS QUITS` |
-| [Weechat](https://weechat.org/) | `/filter add irc_smart_weechat irc.somename.#频道名称 irc_smart_filter *`。**Note:** somename 是你在 Weechat 中定义的 irc 服务器地址。 |
-| [KVIrc](https://www.kvirc.net/) | 参阅[KVIrc 论坛上的信息](http://www.kvirc.ru/forum/?topic=609.0) . |
-| [mIRC](https://www.mirc.com/) | Tools » Options » pick "IRC". Click the "Events..." button. Change the "joins", "parts", "quits", and "nicks" to your desired settings: "In Status" or "Hide" are good options [1](http://web.archive.org/web/20160304201229/http://i.clintecker.com/disable-irc-msgs.html). |
-| [Quassel IRC](https://quassel-irc.org/) | 右键点击聊天窗口，选择 Hide Events » Join/Part/Quit. |
-| [XChat](http://xchat.org/) | Right-click on the tab you want to change. In the submenu of the channelname, there's a toggle-item "Show join/part messages", simply turn this off. Or you type `/set irc_conf_mode 1` [2](http://xchat.org/faq/#q211) to disable the messages throughout the channels. |
+1. 检查你使用的密码是否和 [IRC 验证页面](https://osu.ppy.sh/p/irc) 上的一致。
```
Comment (cl8n) [inline wiki/Community/Internet_Relay_Chat/zh.md:63]: above this, there's a line in EN missing from translation

**Review by cl8n — CHANGES_REQUESTED** [review]

**Comment by cdwcgt** [comment]
I lost line 63 `Try the following:`
I will fix it

File: wiki/Community/Internet_Relay_Chat/zh.md (hunk 8 above)
Comment (cdwcgt) [reply wiki/Community/Internet_Relay_Chat/zh.md:85]: Most of these I think are grammatical reasons.
English uses `connected` directly then ignore the server

Users connected **to osu!Bancho** via osu! client or web site have no prefix.

Comment (cl8n) [reply wiki/Community/Internet_Relay_Chat/zh.md:85]: if it's just a grammar thing and it'd make more sense to readers like this, then it's ok. just making sure it wasn't a mistake

Comment (cdwcgt) [reply wiki/Community/Internet_Relay_Chat/zh.md:85]: Then I don't think there will be any problem.
Many players consider bancho as a server that includes chat

File: wiki/Community/Internet_Relay_Chat/zh.md [hunk 10]
```diff
@@ -1,88 +1,87 @@
...
+- `@`，或者“聊天管理员状态”：用户是聊天室管理员（[GMT（全局管理团队）](/wiki/People/The_Team/Global_Moderation_Team) 或者 [NAT（审核评估团队）](/wiki/People/The_Team/Nomination_Assessment_Team)）
 
-### 我在某些人的名字前看到了 voice 状态，那是什么？
+通过游戏或网站连接到 osu!Bancho 的用户将没有用户名前缀。
 
-voice 状态（名字前面有一个“+”）的用户是通过 IRC 客户端连接的用户，（除了 IRC 管理员，他们的名字前有一个“@”）。
+### 有人给我发送了私信，但是他们并没有在频道用户列表中！
 
-名字前没有前缀的用户是通过游戏连接到 osu! Bancho 的用户。
+他们要么在使用[网页聊天](https://osu.ppy.sh/community/chat)或者使用[osu!(lazer)](https://github.com/ppy/osu)来连接 osu!Bancho。
```
Comment (kj415j45) [inline wiki/Community/Internet_Relay_Chat/zh.md:87]:
```suggestion
他们可能在使用[网页聊天](https://osu.ppy.sh/community/chat)或者[osu!(lazer)](https://github.com/ppy/osu)来连接 osu!Bancho。
```

**Review by kj415j45 — CHANGES_REQUESTED** [review]

**Review by heipizhu4 — APPROVED** [review]

File: wiki/Community/Internet_Relay_Chat/zh.md [hunk 11]
```diff
@@ -1,88 +1,87 @@
...
+默认情况下，许多 IRC 客户端会在用户加入或离开频道时通知您。对于像 osu!Bancho 这样有成千上万的用户频繁登录和退出的服务器，这些消息会刷屏，你可能希望将其隐藏。下表包含常见 IRC 客户端列表以及禁用这些消息的方法：
 
-记住： **不要把你的 IRC 密码交给任何人。**
+| IRC 客户端 | 描述 |
+| :-- | :-- |
+| [HexChat](https://hexchat.github.io/) | 在`设置` -> `首选项` -> `聊天` -> `一般` "隐藏进入/离开信息"（适用于 pre-2.9.6 及以上版本） |
+| [ircII](http://www.eterna.com.au/ircii/) | 输入 `IGNORE * CRAP` |
+| [Irssi](https://irssi.org) | 输入 `/ignore * JOINS PARTS QUITS` |
+| [Weechat](https://weechat.org/) | 输入 `/filter add joinquit * irc_join,irc_part,irc_quit *` |
+| [Konversation](https://konversation.kde.org/) | 在`` |
```
Comment (cl8n) [inline wiki/Community/Internet_Relay_Chat/zh.md:47]: didn't notice last time, this is also missing

**Review by cl8n — DISMISSED** [review]

**Review by kj415j45 — APPROVED** [review]
lgtm

**Review by TPGPL — APPROVED** [review]

**Pull request merged**